flow_py_sdk/client/client.py

```diff
@@ -528,6 +528,8 @@ async def execute_transaction(
 
         """
         log.debug(f"Sending transaction")
+        tx.submit_with_payload_signature()
+        tx.submit_with_envelope_signature()
         result = await self.send_transaction(transaction=tx.to_grpc())
```

Comment:
It would be easier if the user didn't need to do this. I suggest changing these three lines to 
```
result = await self.send_transaction(transaction=tx.to_signed_grpc())
```